subprojects/language-native/src/main/java/org/gradle/language/swift/SwiftBinary.java

```diff
@@ -82,14 +82,7 @@
     /**
      * {@inheritDoc}
      *
-     * @since 4.5
-     */
-    SwiftPlatform getTargetPlatform();
-
-    /**
-     * Returns the Swift language level to use to compile the source files.
-     *
-     * @since 4.6
+     * @since 5.2
      */
-    Provider<SwiftVersion> getSourceCompatibility();
```

Comment:
Should we deprecate this and remove it in 5.3 for the developer already using this property?